### DISCUSSION TITLE
Adding Volume/Issue Numbers to Publication Metadata?

### DISCUSSION THREAD

**Discussion by ideahjournal** [knowledgefutures/pubpub#2847] [Support]
We are interested in assigning volume numbers and issue numbers to individual publications. We are interested in doing this in the publication metadata so it is rendered in the automatic cite function. Is this possible? If so, can anyone provide instructions for how to do this? If not, does anyone have a workaround?

**Comment by gabestein** [comment]
Hello — this is possible using the "Issue" type Collection.
